File: 1897_Redistribute_Characters_to_Make_All_Strings_Equal/main.go
func makeEqual(words []string) bool {
	m := make(map[rune]int)

	for _, word := range words {
		for _, char := range word {
			if _, ok := m[char]; !ok {
				m[char] = 0
			}
			m[char]++
		}
	}

	l := len(words)
	for _, v := range m {
		if v%l != 0 {
			return false
		}
	}
	return true
}

File: 1897_Redistribute_Characters_to_Make_All_Strings_Equal/main.py
class Solution:
    def makeEqual(self, words: List[str]) -> bool:
        chars = ''.join(words)
        charSet = set(chars)

        l = len(words)
        for c in charSet:
            if chars.count(c) % l != 0:
                return False
        return True
